packages/syntax-core/src/RadioButton/RadioButton.module.css

```diff
@@ -1,19 +1,30 @@
 .radioBaseContainer {
   display: flex;
   flex-direction: row;
+  gap: 8px;
   align-items: center;
-  cursor: pointer;
   position: relative;
 }
 
+.radioBaseContainer:focus-within {
+  outline: 2px solid var(--color-base-primary-700);
+}
+
+.stateDisabled {
+  cursor: auto;
+  opacity: 0.3;
+}
+
+.stateEnabled {
+  cursor: pointer;
```

Comment:
I added a `cursor: pointer` to `.radioStyleOverride`. I think we have to consider the enabled/disabled state for it too

Reply:
@JFChoe I used `pointer-events: none` instead